docs/sources/tempo/operations/tempo_cli.md

```diff
@@ -323,8 +322,9 @@ tempo-cli analyse block --backend=local --bucket=./cmd/tempo-cli/test-data/ sing
 
 ## Analyse blocks
 Analyses all blocks in a given time range and outputs a summary of the blocks' generic attributes.
+<!-- Commenting out until vParquet3 is available.
```

Comment:
this doesn't exist in the release-2.2 branch. this will give you a merge failure when it tries to backport